Client/MirObjects/MonsterObject.cs

```diff
@@ -2108,6 +2779,18 @@ public override void DrawEffects()
                             break;
                     }
                     break;
+                    //stupple may need to take out ???
```

Comment:
this put ring round tho monster it USA mir did. i cant remember if the right image if i cord this right i change it few time